index.js

```diff
@@ -88,7 +89,9 @@ module.exports = function(robot) {
     }
 
     if (!queue.isEmpty() && !queue.isCurrent(user)) {
-      // Send DM to next in line if the queue isn't empty and it's not the person who just finished deploying.
+      // Message about the next user to make the queue more obvious
+      res.reply(queue.current().name + ' is up next');
+      // Send message to next in line if the queue isn't empty and it's not the person who just finished deploying.
       notifyUser(queue.current());
```

Comment:
Does it make sense to put the channel notify logic in `notifyUser` as well? It would require passing an additional parameter, but given the other PR, it might make sense? But rule of 3 says you can just still copy paste, :man_shrugging: